server/src/test/java/io/crate/integrationtests/LogicalReplicationITest.java

```diff
@@ -130,6 +131,26 @@ public void test_cannot_drop_user_who_owns_subscriptions() throws Exception {
                     "User 'subscription_owner' cannot be dropped. Subscription 'sub1' needs to be dropped first.");
     }
 
+    @Test
+    public void test_privileges_dont_hide_url_exception() throws Exception {
+        executeOnPublisher("CREATE TABLE doc.t1 (id INT)");
+        createPublication("pub1", false, List.of("doc.t1"));
+
+        String subscriptionOwner = "subscription_owner";
+        executeOnSubscriber("CREATE USER " + subscriptionOwner);
+        executeOnSubscriber("GRANT AL TO " + subscriptionOwner);
+
+        UserLookup userLookup = subscriberCluster.getInstance(UserLookup.class);
+        User user = Objects.requireNonNull(userLookup.findUser(subscriptionOwner), "User " + subscriptionOwner + " must exist");
+        var stmt = String.format(Locale.ENGLISH, "CREATE SUBSCRIPTION sub1 CONNECTION 'crate://localhost:12345/mydb?user=%s&mode=pg_tunnel'" +
+                                                 " publication pub1", user.name());
+        assertThatThrownBy(() -> subscriberSqlExecutor.executeAs(stmt, user))
+            .isExactlyInstanceOf(CreateSubscriptionException.class)
+            .hasMessageContaining(
+                "Database name \"mydb\" is not supported inside the connection string: " +
```

Comment:
I saw a planner test and was going to ask whether it's enough to have it but it's deleted now. 

What was the reason of removal?

Reply:
Because it doesn't actually catch the error, it has to go through the execution stack to reproduce, so I think it's misleading to add a unit test that doesn't cover the case.